test/test_transforms.py

```diff
@@ -1623,9 +1623,9 @@ def _test_transformation(a, t, s, sh):
         _test_transformation(a=0.0, t=(0.0, 0.0), s=1.0, sh=sh)
 
         # Test rotation, scale, translation, shear
-        for a in range(-90, 90, 25):
+        for a in range(-90, 90, 36):
```

Comment:
I'm curious, did the change of values for `a` and `s` actually have an impact on the running time here?

Reply:
Yes it lowered the running times significantly because it decreased the number of executions. 

@vfdev-5 who wrote the original unit-tests advised me on the values we should use to ensure we test correctly the methods.